challenge.rb

```diff
@@ -1,26 +1,38 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
```

Comment:
Redo this challenge please. The goal here is to use methods from Ruby's Enumerable class. Please review the documentation. :)

https://github.com/paircolumbus/MoreEnumerables/blob/master/README.md